image-prod.yaml

```diff
@@ -0,0 +1,55 @@
+- schema_version: 1
+  name: "builder"
+  description: "Golang builder image"
+  version: ""
+  from: "registry.redhat.io/ubi8/go-toolset:1.17.7"
+  modules:
+    repositories:
+      - name: builder
+        path: modules/builder-prod
+    install:
+      - name: builder
+
+  osbs:
+    configuration:
+      container:
+        image_build_method: imagebuilder
+        remote_source:
+          repo: https://github.com/kiegroup/kie-cloud-operator
```

Comment:
leftover?

Reply:
@spolti are you talking about _repo_ and _ref_? They are used by Cachito.

Reply:
Yes.
It is pointing to a specific ref.

how that will be handled and how the right branch will be selected?

will this value be changed on every build?

Reply:
@spolti This is the correct approach FMPOV. When Prod team builds this image, we update the ref with the commit on the top of the branch, same as it is done for Kogito Operator, see https://github.com/kiegroup/kogito-operator/blob/1.13.x/rhpam-image-prod.yaml#L16-L28

Reply:
> When Prod team builds this image, we update the ref with the commit on the top of the branch

ok, that answers my question :)